src/app/features/create-event/event-details-form/event-details-form.component.ts

```diff
@@ -17,6 +17,8 @@ export class EventDetailsFormComponent implements OnInit, OnDestroy {
   private createEventService = inject(CreateEventService);
   private unsubscribe$$ = new Subject<void>();
 
+  categories = this.createEventService.getAllCategories();
+
```

Comment:
podobnie jak w html `categories` brakuje $, ponieważ to observable `categories$`

Reply:
faktycznie, jakoś mi umkneło